src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -353,7 +355,13 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
-  def createBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = dataSource.inTransaction { dataAccess =>
+  def deleteBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
+    // delete actual project in google-y way, then remove from Rawls DB
+    DBIO.from(gcsDAO.deleteProject(projectName))
```

Comment:
the result of this future is lost. This function should probably look like
gcsDAO.deleteProject(projectName) flatMap { _ => unregisterBillingProject(projectName) }